.github/workflows/3-multi-runtime.yaml

```diff
@@ -58,6 +58,13 @@ jobs:
         uses: actions/checkout@v3
       - name: "Setup"
         run: ./tools/github/setup.sh
+      - name: Maximize free space
```

Comment:
Based on this workaround: https://github.com/actions/runner-images/issues/2840#issuecomment-790492173

Reply:
Since multi-runtime tests passed with this change, I applied this change to all workflows.